workshops/views.py

```diff
@@ -817,23 +817,27 @@ def search(request):
                 sites = Site.objects.filter(
                     Q(domain__contains=term) |
                     Q(fullname__contains=term) |
-                    Q(notes__contains=term))
+                    Q(notes__contains=term)) \
+                    .order_by('fullname')
```

Comment:
It's a QuerySet method https://docs.djangoproject.com/en/1.8/ref/models/querysets/#order-by

The slowdown shouldn't be noticeable, because databases are good at sorting.